### DISCUSSION TITLE
【灌水】绳网等级竟然和repository绑定

### DISCUSSION THREAD

**Discussion by March-777th** [share121/inter-knot#1512] [其他]
###### +3

**Comment by capacitor1** [comment]
赞同
+3

**Comment by user1-cloud** [comment]
# 不赞同
那样会出现大量水帖的

**Comment by Dave-12138** [comment]
因为如果存等级，那就需要一个数据库，而现在的绳网是没有后端的
如果不存，而是每次都走api获取发布讨论个数——总感觉这样会有很大的网络性能开销

**Comment by capacitor1** [comment]
> Reply::不赞同

repository绑定的话，也可以随便fork一堆东西。。或者建一堆空仓库

防不住想水的

但是真心要做一个积分系统，那太复杂了。。
目前比较贴切的积分系统就是bitcomet里的按照上传总量和在线时长来积分的

**Comment by cpu644** [comment]
> 目前比较贴切的积分系统就是bitcomet里的按照上传总量和在线时长来积分的
@capacitor1  像一些大点的discord服务器那样搞个后端存储上传量/用户  然后基于上传量加个冷却（eg. 每xxx分钟/小时的第一个帖子+xxx积分）？不过还是有水的可能性 只能说大佬辛苦了

**Comment by lllliufeng** [comment]
我们可以用十五字大法罢（
